stable/kong/templates/ingress-admin.yaml

```diff
@@ -17,6 +18,14 @@ metadata:
     {{- end }}
 spec:
   rules:
+    {{- if eq $hosts_count 0 }}
+    - http:
+        paths:
+          - path: {{ $path }}
+            backend:
+              serviceName: {{ $serviceName }}-admin
+              servicePort: {{ $servicePort }}
+    {{ else -}}
```

Comment:
Have a look at the `.ci` directory and edit any two files to introduce integration tests for this change please. In one, you could test the new behavior and in the other, the older behavior.
This will be a blocker for the PR.

Also, please rebase on the latest master branch of this repository.

Reply:
Done & done. I added new files for the ci tests as I didn't think it made sense to add them to any of the current ones, as they all seemed to be for a specific case, but if you would rather I do it that way then happy to change it!